src/utils/SVG.cpp

```diff
@@ -210,7 +285,45 @@ void SVG::writePolygon(ConstPolygonRef poly, Color color, float stroke_width)
     int i = 0;
     for (Point p1 : poly)
     {
-        if (color == Color::RAINBOW)
+        if (color.color == Color::RAINBOW)
```

Comment:
Perhaps it's good to document what this does.

This seems to select one of 11 rainbow colours depending on the number of vertices in the polygon.

Reply:
ok